src/rules/bn.toml

```diff
@@ -0,0 +1,52 @@
+
+# Replacements which are run before sentences are split
+replacements = [
+  ["ডাঃ","ডাক্তার"],
+  ["ডা.","ডাক্তার"],
+  ["ড.","ডাক্তার"],
+  ["নাং", "নম্বর"],
+  ["ন.", "নম্বর"],
+  ["নাং.", "নম্বর"],
+  ["নং", "নম্বর"],
+  ["মো.", "মোহাম্মদ"],
+  ["মোঃ", "মোহাম্মদ"]
+]
+
+min_trimmed_length = 3
+min_word_count = 2
+max_word_count = 20
+min_characters = 5
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = false
+broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;"]
+
+allowed_symbols_regex = "[॰ঁংঃঅআইঈউঊঋঌএঐওঔকখগঘঙচছজঝঞটঠডঢণতথদধনপফবভমযরলশষসহ়ঽািীুূৃৄেৈোৌ্ৎৗয়ৠৡ৤০১২৩৪৫৬৭৮৯ৰৱ৳৹৺ਂ!\"',-\\./:;\\?`।„“‚‘’ ]"
+
+disallowed_symbols = [
```

Comment:
This can now be completely removed

Reply:
Is it okay to keep digits in the allowed_symbols_regex ?

Reply:
Oh, good question! We don't allow numbers in general, as it's not necessarily clear how to pronounce them. 2022 could for example be "two-zero-two-two", "twenty-twenty-two" or "two thousand and twenty two". Is that the same for Bengali? If so, let's not allow numbers.

Reply:
Got it my bad. I will update the rules to exclude numbers